targets/unity-v2/source/ExampleTestProject/Assets/Testing/Tests/Shared/Uunit/UUnitTestContext.cs

```diff
@@ -53,6 +53,10 @@ public void AttemptRetry()
 
         public void EndTest(UUnitFinishState finishState, string resultMsg)
         {
+            // Add a random waiting jitter to each test end to prevent multiplatform parallel tests from throttling the test title.
+            Random rand = new Random();
+            System.Threading.Thread.Sleep(rand.Next(250));
```

Comment:
250 milliseconds is not much when throttling is only recalculated each second. Consider something in the 1-3 second range.

Reply:
Okay, I was noticing a hiccup in the unity UI when I made it 1 second, but it may be what we have to deal with to prevent the parallelism issue.